### DISCUSSION TITLE
19.0 release date?

### DISCUSSION THREAD

**Discussion by chazeh1** [Stellarium/stellarium#591] [Q&A] [answered]
When is the schedule for 19.0 release and what are the proposed changes compared to version 18.3?

**Comment by alex-w** [comment, marked as answer]
Release of version 0.19.0 scheduled due by March 20, 2019 - https://github.com/Stellarium/stellarium/milestones

List of changes is not complete yet, but we will update them every weekend due release of weekly builds (link Beta at stellarium.org).

**Comment by chazeh1** [comment]
Thank you!
